iModelCore/ECPresentation/Tests/NonPublished/Integration/Hierarchies/HierarchyIntegrationTests.cpp

```diff
@@ -10530,6 +10717,53 @@ TEST_F(RulesDrivenECPresentationManagerNavigationTests, Grouping_PropertyGroup_D
     VerifyNodeInstance(rules->GetRuleSetId(), *childrenNodes[1], *instance2);
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsitest
++---------------+---------------+---------------+---------------+---------------+------*/
+DEFINE_SCHEMA(Grouping_PropertyGroup_GuidProperty, R"*(
+    <ECEntityClass typeName="Element">
+        <ECProperty propertyName="GuidProp" typeName="binary" extendedTypeName="BeGuid" />
+    </ECEntityClass>
+)*");
+TEST_F(RulesDrivenECPresentationManagerNavigationTests, Grouping_PropertyGroup_GuidProperty)

```

Comment:
how is this test different from `Grouping_PropertyGroup_CreatesGroupForSingleItem_GuidProperty`?

Reply:
removed